binder/src/main/java/io/grpc/binder/util/BindServiceFlags.java

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright 2021 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.frameworks.client.data.android.binder;
```

Comment:
Please fix :) Ditto for BindServiceFlagsTest.

FWIW, we'll probably want to name the util package `internal` if users shouldn't poke at it at all. But that can happen later. If they are supposed to use things like this directly, then that's fine.